docs/src/utils/segment-to-name.js

```diff
@@ -0,0 +1,8 @@
+// Return the word with only the first letter uppercased
```

Comment:
I would suggest using JSDoc

Reply:
Yeah, this is something. I've been looking around the internet today and found [`ibm/sveld`](https://github.com/IBM/sveld) which can generate TS type definitions from JSDoc. We may consider it

Reply:
Cool! It's less than a month old, but I say we go for it